config/events/events.cwt

```diff
@@ -164,10 +167,21 @@ event = {
 
     ## cardinality = 0..1
     theme = enum[event_themes]
-
+    ## cardinality = 0..1
+    override_background = {
+        ## cardinality = 0..1
+        event_background = enum[event_backgrounds]
+    }
 }
 
 single_alias[event_portrait] = {
-
+    character = scope[character]
+    ## cardinality = 0..1
+    animation = enum[animations]
+    ## cardinality = 0..1
+    triggered_animation = {
+        trigger = single_alias_right[trigger_clause]
+        animation = enum[animations]
+    }
```

Comment:
Does order matter for CWTools? Because then line 180 should come below 185.

Reply:
No, because afaik order never matters in the game engine either. Suggestions are ordered by appropriate, then usage, then alphanumeric